Tools/CMake/AMReX_Config.cmake

```diff
@@ -48,9 +48,14 @@ function (configure_amrex)
    # Moreover, it will also enforce such standard on all the consuming targets
    #
    set_target_properties(amrex PROPERTIES CXX_EXTENSIONS OFF)
-
    target_compile_features(amrex PUBLIC cxx_std_11)  # minimum: C++11
 
+   if (ENABLE_CUDA AND (CMAKE_VERSION VERSION_GREATER_EQUAL 3.17) )
+       set_target_properties(amrex PROPERTIES CUDA_EXTENSIONS OFF)
+       target_compile_features(amrex PUBLIC cuda_std_11)  # minimum: C++11
```

Comment:
Oh dang, that's a CMake 3.17+ feature. Needs guard.
https://gitlab.kitware.com/cmake/cmake/-/issues/20559

Reply:
This change was already committed 6 days ago, was it not?

Reply:
Yes, I was just a confused fellow.